var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -30,8 +30,10 @@ class Zee(CMakePackage):
     variant('codechecks', default=False,
             description='Perform additional code checks like ' +
                         'formatting or static analysis')
+    depends_on('boost')
```

Comment:
Just to understand: are we using boost explicitly in the zee core or just got tests?

Reply:
I am using boost timer explicitely

Reply:
Ok (that's seems bit heavy dependency for timer though!)

Reply:
> Ok (that's seems bit heavy dependency for timer though!)

I will use `std::chrono` library instead, but I like to have boost available if needed.